spec/requests/invites_controller_spec.rb

```diff
@@ -203,6 +203,52 @@
     end
   end
 
+  context '#create_multiple_use_invite_link' do
+    it 'requires you to be logged in' do
+      post "/invites/link.json", params: {
+        max_redemptions_allowed: 5
+      }
+      expect(response).to be_forbidden
+    end
+
+    context 'while logged in' do
+      it "fails for non-admin users" do
+        sign_in(Fabricate(:trust_level_4))
+        post "/invites/link.json", params: {
+          max_redemptions_allowed: 5
+        }
+        expect(response).to be_forbidden
+      end
+
+      it "allows admins to invite to groups" do
+        group = Fabricate(:group)
+        sign_in(Fabricate(:admin))
+
+        post "/invites/link.json", params: {
+          max_redemptions_allowed: 5,
+          group_names: group.name
+        }
+
+        expect(response.status).to eq(200)
+        expect(Invite.multiple_use_invites.last.invited_groups.count).to eq(1)
+      end
+
+      it "allows multiple group invite" do
+        Fabricate(:group, name: "security")
+        Fabricate(:group, name: "support")
+        sign_in(Fabricate(:admin))
+
+        post "/invites/link.json", params: {
+          max_redemptions_allowed: 5,
+          group_names: "security,support"
```

Comment:
Hmm why do we allow group names to be used? A group can be renamed so the `Group#name` isn't as reliable as `Group#id`.

Reply:
We have been allowing group names ever since we started supporting adding group access for invites.